templates/index.html

```diff
@@ -1,14 +1,20 @@
 {% extends "_layout.html" %}
 
 {% block content %}
+<div>
+    <div class="p-4">
+        <h1 class="display-2 fw-bold bold-orange-title">Welcome To <span class="white-title">TRAINFIT</span></h1>
+        <p class="lead fs-4 description">TrAInFit is an innovative AI-supported health application designed
+            to revolutionize the way you manage your well-being. With a seamless integration
+            of cutting-edge technology and user-friendly features, HealthTrackr aims to empower
```

Comment:
Reverse this please, its not supposed to say healthtracker im assuming just a typo on ur end